File: hw/src/main/java/com/sumian/app/improve/guideline/bean/Guideline.java
package com.sumian.app.improve.guideline.bean;

import android.support.annotation.DrawableRes;
import android.support.annotation.StringRes;

/**
 * Created by sm
 * on 2018/3/22.
 * desc:
 */

public class Guideline {

    public @StringRes
    int h1Label;
    public @StringRes
    int h2Label;
    public @DrawableRes
    int iconId;
    public int indicatorPosition;

    @Override
    public String toString() {
        return "Guideline{" +
            "h1Label='" + h1Label + '\'' +
            ", h2Label='" + h2Label + '\'' +
            ", iconId=" + iconId +
            ", indicatorPosition=" + indicatorPosition +
            '}';
    }
}

File: patient/src/main/java/com/sumian/sd/examine/guide/bean/Guideline.kt
package com.sumian.sd.examine.guide.bean

import androidx.annotation.DrawableRes
import androidx.annotation.StringRes

/**
 * Created by sm
 * on 2018/3/22.
 * desc:
 */
class Guideline {
    @StringRes
    var h1Label = 0

    @StringRes
    var h2Label = 0

    @DrawableRes
    var iconId = 0
    var indicatorPosition = 0
    override fun toString(): String {
        return "Guideline{" +
                "h1Label='" + h1Label + '\'' +
                ", h2Label='" + h2Label + '\'' +
                ", iconId=" + iconId +
                ", indicatorPosition=" + indicatorPosition +
                '}'
    }
}